pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/model/dsm/DsmNotificationEventType.java

```diff
@@ -62,5 +62,16 @@ public enum DsmNotificationEventType {
     /**
      * Message sent by DSM on stool received event.
      */
-    STOOL_RECEIVED
+    STOOL_RECEIVED,
+
+    /**
+     * Message sent by DSM on tissue received event.
+     */
+    TISSUE_RECEIVED,
```

Comment:
iirc DSM is not sending this anymore. @SimoneMaiwald knows better

Reply:
I added for future just incase. 
I am not using it as of now